.github/dependabot.yaml

```diff
@@ -16,3 +16,10 @@ updates:
       interval: "daily"
     labels:
       - "dependencies"
+    # Not yet supported. See <https://github.com/dependabot/dependabot-core/issues/4009>.
+    # versioning-strategy: "increase-if-necessary"
+    ignore:
+      - dependency-name: "serde"
+        update-types: ["version-update:semver-minor", "version-update:semver-patch"]
```

Comment:
The commit message says we're ignoring major versions for serde but we're ignoring minor ones ;)

Reply:
🤦